src/org/usfirst/frc/team2335/robot/subsystems/Ultrasound.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Ultrasound extends Subsystem
+{
+	Ultrasonic backUltra;
+	Ultrasonic frontUltra;
+
+	public Ultrasound()
+	{
+		backUltra = new Ultrasonic(Robot.BACK_PING, Robot.BACK_ECHO);		
+		frontUltra = new Ultrasonic(Robot.FRONT_PING, Robot.FRONT_ECHO);
+		
+		backUltra.setAutomaticMode(true);
+		frontUltra.setAutomaticMode(true);
+	}
```

Comment:
Just to clarify, one of the Ultras is the sender and the other is the reciever, right?